e3sm_diags/derivations/derivations.py

```diff
@@ -109,6 +109,10 @@
         ("pr",): lambda pr: qflxconvert_units(rename(pr)),
         ("PRECC", "PRECL"): lambda precc, precl: prect(precc, precl),
         ("sat_gauge_precip",): rename,
+        ("precip_liq_surf_mass_flux", "precip_ice_surf_mass_flux"): prect,  # EAMxx
+        ("precip_total_surf_mass_flux",): lambda pr: convert_units(
+            rename(pr), target_units="mm/day"
+        ),  # EAMxx
```

Comment:
Should we considering separating EAMxx variables out to another derivations dictionary so we don't need to comment `# EAMxx`?

This can be done through #716.

Reply:
let's keep the comments for now and we can consider separation in the future. We probably want to separate variables from multiple obs sources as well.